skimage/feature/_hog.py

```diff
@@ -29,12 +30,15 @@ def hog(image, orientations=9, pixels_per_cell=(8, 8),
         Number of cells in each block.
     visualise : bool, optional
         Also return an image of the HOG.
-    normalise : bool, optional
+    transform_sqrt : bool, optional
         Apply power law compression to normalise the image before
-        processing.
+        processing. DO NOT use this if the image contains negative values.
```

Comment:
Add some text here explaining the effect of "power law" compression, what "power law compression" is, why you should do it, etc.  This can also be done in the "Notes" section if you prefer, with a reference to that section here.